public/locales/de/translations.json

```diff
@@ -0,0 +1,49 @@
+	"goals.stabilization.title": "Stabilisierung",
+	"goals.stabilization": "Benutzerfreundliche Funktionen wie die 'Nix v3 CLI' werden standardmäßig aktiviert. Nix 'Flakes' werden bereits zu häufig verwendet, um sie zu entfernen oder zu ändern, stattdessen werden diese als 'flake v0' in ihrem aktuellen Zustand vereinheitlicht. Zukünftige Arbeiten werden von einer Special Interest Group gesteuert.",
+	"goals.infra.title": "Infrastruktur",
+	"goals.infra": "Wir werden die Tools und die Infrastruktur aufbauen und betreuen, welche erforderlich sind, um das Ökosystem stabil zu halten, einschließlich 'Continous Integration' und 'Binary Caching'.",
+	"goals.education.title": "Dokumentation",
+	"goals.education": "Einheitliche Dokumentation, Anleitungen, Leitfäden und Beispiele sorgen für ein besseres Onboarding-Erlebnis für Neulinge und einen schnelleren Iterationszyklus für Experten.",
+
+	"roadmap.title": "Der Fahrplan",
+	"roadmap.soft_fork.title": "Soft Fork",
+	"roadmap.soft_fork.1": "Während der Anfangsphase wird eine einfache Ad-hoc-Managementstruktur genutzt. Die erklärten Werte, Ziele und Roadmap des Projekts dazu allen Mitwirkenden die Sicherheit zu geben, dass sich die Arbeit, zu der sie sich verpflichtet haben, nicht verändern wird.",
+	"roadmap.soft_fork.2": "In dieser Phase wird die erste Abspaltung von Nix und NixPkgs umgesetzt und Mitwirkende erhalten Rollen für die Paketwartung und das Synchronisieren von Änderungen von 'upstream'. Wir beabsichtigen diese Phase zu verlassen, sobald genügend Mitwirkende beigetreten sind, damit das Projekt in die 'Hard Fork' Phase übergehen kann.",
+	"roadmap.hard_fork.title": "Hard Fork",
+	"roadmap.hard_fork.1": "Wie in der 'Soft Fork'-Phase wird weiterhin eine Ad-hoc-Managementstruktur zum Einsatz kommen, es wird jedoch mit der Einrichtung von 'Committees', 'Special Interest Groups' und 'Working Groups' begonnen. Demokratische Wahlen werden erst nach Abschluss dieser Phase durchgeführt um die nötigen Arbeiten zu erleichtern.",
+	"roadmap.hard_fork.2": "Zu diesem Zeipunkt werden wir ausreichend Mitwirkende haben, um NixPkgs (oder angemessene Teile davon) und Nix selbst sicher zu verwalten. Wir werden damit beginnen, Abweichungen vom 'Upstream Repository' zuzulassen und Änderungen an der Repository-Struktur zur Verbesserung der Entwicklung einzuführen.",
+	"roadmap.hard_fork.3": "Eine erste öffentliche Version des Paket-Repositorys wird erstellt und die Umbenennung `nix` auf `aux` gestartet, um Namesverwechselungen oder Fehler zu reduzieren.",
+	"roadmap.organization.title": "Strukturierung",
+	"roadmap.organization.1": "Aus dem Paket-Repository werden 'sets' extrahiert, und die Verwaltung an spezialisierte 'Sepcial Interest Groups' übergeben um langfristig eine einfachere Verwaltung zu ermöglichen. Beispielsweise werden `pythonPackages` in ein separates Repository verschoben, dessen Verwaltung in die Verantwortung der 'Python SIG' fällt.",
```

Comment:
suggestion: Maybe 'Paket-Gruppen' instead of sets?

Reply:
I was reading 'sets' as a specific technical term. But given this is a big picture overview using a more descriptive term makes a lot of sense to me.